### PR TITLE
Module 3

### PR DESCRIPTION
**Pull request by homergit**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -948,17 +948,17 @@ <h3 class="subtitle is-5">Don't hesitate, ask anything</h3>
         </div>
 
         <div class="field">
-          <label for="email" class="label" required>Email</label>
+          <label for="email" class="label">Email</label>
           <div class="control has-icons-left has-icons-right">
-            <input id="email" class="input is-danger" type="email" placeholder="Email input" value="hello@">
+            <input id="email" class="input is-danger" type="email" placeholder="Email input" value="test@gmail.com">
             <span class="icon is-small is-left">
               <i class="fas fa-envelope"></i>
```
Comment (IrinaShilina) [inline index.html:955]: You should apply `aria-hidden` to the icons.

File: index.html [hunk 2]
```diff
@@ -918,7 +918,7 @@ <h4>How to make tiles wrap with CSS?</h4>
     </div>
   </section>
 
-  <section id="contact" class="section has-background-light">
+  <section id="contact" class="section has-background-light" aria-label="our contacts">
```
Comment (IrinaShilina) [inline index.html]: We usually use ‘aria-labelledby’ and point it to the section's title to tag to avoid the repetition

**Review by IrinaShilina — COMMENTED** [review]
Please apply accessibility attributes to the modals: https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html

File: index.html (hunk 1 above)
Comment (homergit) [reply index.html:955]: added, thanks!

File: index.html (hunk 2 above)
Comment (homergit) [reply index.html]: Changed to aria-labelledby for all sections on the page
Thank you

**Pull request merged**